src/chainparams.cpp

```diff
@@ -352,7 +352,7 @@ class SigNetParams : public CChainParams {
         consensus.vDeployments[Consensus::DEPLOYMENT_TAPROOT].min_activation_height = 0; // No activation delay
 
         // message start is defined as the first 4 bytes of the sha256d of the block script
-        CHashWriter h(SER_DISK, 0);
```

Comment:
nit: what's the benefit of writing this `HashWriter h{};` instead of `HashWriter h`?

Reply:
The behaviour is the same with or without `{}`. However, the (style) benefit of always using `{}` is that it can be used consistently in code. For example, you couldn't write `Function(HashWriter)` to construct a temporary.